robot/impl/local_robot_test.go

```diff
@@ -801,6 +801,10 @@ func TestStopAll(t *testing.T) {
 		]
 	}
 	`, model.String())
+	defer func() {
```

Comment:
I added them now because I NEEDED them to be able to run `go test ... -count 5` in order to hit/find/prove the race sometimes. But, I only did them here. I didn't try to fix it up across the whole RDK.

Reply:
Sounds good; I don't think it's necessary to add across the RDK.